examples/crossmodel_bundle.py

```diff
@@ -0,0 +1,84 @@
+"""
+This example:
+
+1. Connects to the current controller
+2. Creates two models for consuming and offering
+2. Deploys a charm and waits until it reports itself active
+3. Creates an offer
+4. Lists the offer
+5. Deploys a charm and adds relation to the offering url
+6. Destroys the units and applications
+
+"""
+import time
+from logging import getLogger
+from pathlib import Path
+
+from juju import loop
+from juju.client.connection import facade_versions
+from juju.controller import Controller
+
+log = getLogger(__name__)
+
+
+async def main():
+    controller = Controller()
+    print("Connecting to controller")
+    await controller.connect(specified_facades=facade_versions("Bundle", [1, 2, 3]))
```

Comment:
? does the end user need to specify these facade versions?

Reply:
Nope, I'll remove it.